Producer/Producer.Droid/Framework/Fragments/SettingsFragment.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Android.Views;
+using Android.OS;
+using Android.Content;
+using Producer.Domain;
+using Android.Preferences;
+using Android.Support.V4.Content;
+using SettingsStudio;
+using System.Collections.Generic;
+
+namespace Producer.Droid
+{
+	public class SettingsFragment : PreferenceFragment
+	{
+		static readonly string [] Preferences = { 
```

Comment:
Roberto merge master into this branch then change this to use [`SettingsKeys.VisibleSettings`](https://github.com/technicalpoets/producer/blob/master/Producer/Producer.Shared/Settings/SettingKeys.cs#L35)